nodepool/nl02.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -17,6 +17,8 @@ labels:
     max-ready-age: 600
   - name: centos-8-1vcpu
     max-ready-age: 600
+  - name: centos-8-8vcpu-8gb
```

Comment:
centos-8-8vcpu should be okay

Reply:
I added the -8gb to have a more meaningful name. Should I remove it?

Reply:
yah, lets keep in sync with current layout, this is because reach provider doesn't have same cpu / ram options. So we defaulted to vcpu for tracking size

Reply:
I see, I will adjust that.